github/plugin.spec.yaml

```
@@ -1142,6 +1142,70 @@ actions:
         description: URL
         type: string
         required: false
+  create_issue_comment:
+    title: Create Issue Comment
+    description: Create an issue commentt
```

Typo
```suggestion
    description: Create an issue comment
```
And then run `make regenerate` so the changes reflect in the schema files